builder/ghc-for-component-wrapper.nix

```diff
@@ -27,7 +27,7 @@ let
     else ghc;
 
 in runCommand "${componentName}-${ghc.name}-env" {
-  preferLocalBuild = true;
+  preferLocalBuild = false;
```

Comment:
This is sad. It's much nicer having things like this build locally. I don't quite understand the issue: surely `preferLocalBuild` won't make it build locally if your system is the wrong one?

Reply:
That this is necessary seems like a Nix bug.